### PR TITLE
Created ship class

### PR DESCRIPTION
**Pull request by amsmyth1**
Created ship class and test code
Created readable attributes: name, length, health
Created methods: hit, sunk

### CODE REVIEW

File: test/ship_test.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/ship'
+
+class ShipTest < MiniTest::Test
+
```
Comment (jbranham1) [inline test/ship_test.rb]: Can you take empty line off of line 6?

**Review by jbranham1 — CHANGES_REQUESTED** [review]
Overall looks super great! I did have one comment for a fix.

File: test/ship_test.rb (hunk 1 above)
Comment (amsmyth1) [reply test/ship_test.rb]: Empty line 6 removed.

**Review by jbranham1 — APPROVED** [review]
Looks great!

**Pull request merged**